CHANGELOG.md

```diff
@@ -1,18 +1,14 @@
-# NEXT RELEASE
+# 10.8.0 Release notes
 
 ### Enhancements
 * Added the functionality to delete files for a given SharedRealm, unlocking  ([realm-dotnet#386](https://github.com/realm/realm-dotnet/issues/386)).
```

Comment:
This is missing a reference to an issue or PR. I would also suggest to write what the method is that was added.